aws/resource_aws_vpc_endpoint_test.go

```diff
@@ -179,8 +179,7 @@ func TestAccAWSVpcEndpoint_gatewayWithRouteTableAndPolicy(t *testing.T) {
 func TestAccAWSVpcEndpoint_gatewayPolicy(t *testing.T) {
 	var endpoint ec2.VpcEndpoint
 	// This policy checks the DiffSuppressFunc
-	policy1 := `
-{
+	policy1 := `{
```

Comment:
It appears this policy is being directly injected into heredoc syntax so either the newlines need to be present here or in the test configuration:

```
=== CONT  TestAccAWSVpcEndpoint_gatewayPolicy
TestAccAWSVpcEndpoint_gatewayPolicy: resource_aws_vpc_endpoint_test.go:217: Step 1/3 error: terraform failed: exit status 1
stderr:
Error: Invalid expression
on config241249749/terraform_plugin_test.tf line 15, in resource "aws_vpc_endpoint" "test":
15:   policy       = <<POLICY{
Expected the start of an expression, but found an invalid expression token.
--- FAIL: TestAccAWSVpcEndpoint_gatewayPolicy (1.55s)
```

Reply:
```
--- PASS: TestAccAWSVpcEndpoint_gatewayPolicy (70.16s) (GovCloud)
--- PASS: TestAccAWSVpcEndpoint_gatewayPolicy (61.74s) (commercial)
```